zebra-state/src/service/finalized_state/zebra_db/block/tests/snapshots/address_balances@testnet_1.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: zebra-state/src/service/finalized_state/zebra_db/block/tests/snapshot.rs
+expression: stored_address_balances
+---
+[
+  ("t2UNzUUx8mWBCRYPRezvA363EYXyEpHokyi", AddressBalanceLocation(
```

Comment:
This address matches the equivalent `zcashd` test:
https://github.com/zcash/zcash/blob/49ffee3f20b972dc3aa75e422c67523251cf088b/src/gtest/test_foundersreward.cpp#L127